tests/test_utils.py

```diff
@@ -94,3 +98,94 @@ def test_fill_in_from_config(self, mocker):
             cli_utils.fill_in_from_config(
                 "jsonld_none", jsonld_none, mock_keys_invalid
             )
+
+
+class FakeResponse:
+    status: int
+    data: bytes
+
+    def __init__(self, *, data: bytes):
+        self.data = data
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, type, value, traceback):
+        pass
+
+    def read(self):
+        self.status = 200 if self.data is not None else 404
+        return self.data
+
+    def close(self):
+        pass
+
+
+class TestIOUtils:
+
+    def test_json_load(self, tmpdir, mocker):
+
+        json_file = tmpdir.join("example.json")
+        json_file.write_text(json.dumps([
+            {'k1': 'v1'},
+            {'k2': 'v2'}
+        ]), encoding="utf-8")
+
+        with open(json_file, encoding='utf-8') as f:
+            expected = json.load(f)
+
+        result = io_utils.load_json(str(json_file))
+
+        assert result == expected
+
+        mock_urlopen = mocker.patch("urllib.request.urlopen",
+                                    return_value = FakeResponse(data=json.dumps([
+                                        {'k1': 'v1'}, 
+                                        {'k2': 'v2'}]
+                                    ))
+                                    )
+        
+        res = io_utils.load_json("http://example.com")
+        assert res == [
+            {'k1': 'v1'},
+            {'k2': 'v2'}
+        ]
+
+        assert mock_urlopen.call_count == 1
```

Comment:
[Nit:](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.) You might want to split this test function into two: `test_json_load_local()` and `test_json_load_online()`. Not required.